src/components/dialogs/InvalidNrDialog.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <v-dialog v-model="dialog" persistent :attach="attach" max-width="60%">
```

Comment:
I was initially going for a standard set of widths for dialogs. Not sure if a percentage works best here. FWIW

Reply:
I went with this smaller size so it was more similar to the Manage business dialogs which was referenced in the designs. I think the idea is have this mini flow as close to the Manage business 'Add Name Request' flow as possible, dialogs included.

Reply:
OK, that's reasonable. You might want to make it a fixed width though (not % as that's screen-size-dependent).

Reply:
Great point!